components/footers/PageFooter.vue

```diff
@@ -41,19 +41,31 @@
             Social Media
           </h2>
           <ul>
-            <li><a class="footer-column__link" href="https://github.com/Qiskit" target="_blank" rel="noopener">GitHub</a></li>
-            <li><a class="footer-column__link" href="https://qiskit.slack.com/" target="_blank" rel="noopener">Slack</a></li>
-            <li><a class="footer-column__link" href="https://twitter.com/Qiskit" target="_blank" rel="noopener">Twitter</a></li>
-            <li><a class="footer-column__link" href="https://medium.com/Qiskit" target="_blank" rel="noopener">Medium</a></li>
-            <li><a class="footer-column__link" href="https://www.youtube.com/Qiskit" target="_blank" rel="noopener">YouTube</a></li>
-            <li><a class="footer-column__link" href="https://quantumcomputing.stackexchange.com/questions/tagged/qiskit" target="_blank" rel="noopener">Stack Exchange</a></li>
+            <li v-for="socialMedia in socialMediaList" :key="socialMedia.label" >
+              <a class="footer-column__link" :href="socialMedia.url" target="_blank" rel="noopener">
+                {{ socialMedia.label }}
+              </a>
+            </li>
           </ul>
         </section>
       </div>
     </div>
   </footer>
 </template>
 
+<script lang="ts">
+import Vue from 'vue'
+import { Component, Prop } from 'vue-property-decorator'
+
+import { orderedQiskitElements, orderedSocialMedia } from '~/constants/links'
+
+@Component
+export default class extends Vue {
+  @Prop({ type: Array, default: () => orderedQiskitElements }) elements
```

Comment:
If I did not understand wrongly, you use properties to expose the sorted lists of elements and social media in the template. However, properties are intended to be used as the public HTML API of the component. Vue components have an specific way of exposing data to the component itself without making it part of the public API. You simply define them inside the class so you can do:
```
export default class extends Vue {
  elements = orderedQiskitElements
  ...
}
```
This is explained in the [Typescript documentation, when using the Class API](https://typescript.nuxtjs.org/cookbook/components/).